InnerEye/ML/Histopathology/models/deepmil.py

```diff
@@ -105,7 +105,11 @@ def get_classifier(self) -> Callable:
 
     def get_loss(self) -> Callable:
         if self.n_classes > 1:
-            return nn.CrossEntropyLoss(weight=self.class_weights)
+            if self.class_weights is not None:
+                class_weights = self.class_weights.float()
+            else:
+                class_weights = self.class_weights
```

Comment:
I find it easier to read if there are no negations (swap if/else)- you are anyway handling both cases.

Reply:
agreed, done